README.md

```diff
@@ -77,13 +79,97 @@ In the sub-folders for plots that intersected with the orthomosaic, there will b
 The CSV files both contain much of the same information.
 The file with "geostreams" in its name can be uploaded to TERRAREF's Geostreams database.  
 
+### Canopy Cover: OpenDroneMap and plot boundaries <a name="opendm_can_shp" />
+The following steps are used when wanting to use OpenDroneMap (ODM) to create the Orthomosaic image that's then used to create the canopy cover values.
+As with the [previous example](#om_can_shp) we will be listing the steps and then providing an example.
+
+_NOTE_: the SciF Docker image uses _Docker siblings_ to run the OpenDroneMap application.
+If you are not comfortable with this approach, you shouldn't use this workflow.
```

Comment:
* [ ] please clarify can this be stated in a way that doesn't depend on the reader's 'comfort'? Like, 'the docker sibling approach has (security? performance?) costs'. Or, how does this actually impact what the user is doing?
* [ ] link "Docker siblings" to documentation.

Reply:
Documentation has been updated

Reply:
the comment "If you are not comfortable with this approach, you shouldn't use this workflow." seems off-putting

Reply:
nevermind. I see this view of the code is out of date